docs/quickstart.md

```diff
@@ -26,9 +26,11 @@ VSCode Remote Containers (Dev Container). To run locally:
 3. Change directory to: `cd carbon-aware-sdk/src/CarbonAware.CLI/src`
 4. If you have a WattTime account registered (or other data source) - you will
    need to configure the application to use them. By default the SDK will use a
-   pre-generated JSON file with random data. To configure the application, you
-   will need to set up specific environment variables or modify
-   `appsetttings.json` inside of `src/CarbonAware.WebApi/src` directory.
+   pre-generated JSON file with random data. This random data is meant to make
+   it easier to get started with the SDK and doesn't
+   represent actual Carbon data. To configure the application,
+   you will need to set up specific environment variables or modify
+   `appsettings.json` inside of `src/CarbonAware.WebApi/src` directory.
    Detailed information on configuration can be found in the `GettingStarted.md`
```

Comment:
These will need updating... they actually point to the wrong location already somehow... I can do the updates, just a comment here so I don't forget.

Reply:
We might end up with some rogue references that pop up but we can fix them as they turn up.